composer.json

```diff
@@ -19,7 +19,7 @@
     },
     "require": {
         "php": "^5.3.9|^7.0",
-        "imagine/Imagine": "^0.6.3,<0.7",
+        "imagine/Imagine": "^0.6.3|^0.7.0,<0.8",
```

Comment:
I realize; I believe the current syntax was used to provide clarity, as even before the change when it was `^0.6.3,<0.7`, the `<0.7` was unnecessary. It's important to note that the rule you mentioned *only* applies for fully enumerated versions, so it may be a good idea to retain the upper constraint so when someone comes along and changes it to, say `^0.7`, we don't open the floodgates.

There are a bunch of ways to write this, such as `^0.6.3|^0.7.0,<0.8`, `>=0.6.3,^0.6,<0.8`, `>=0.6.3,<0.8` or `~0.6.3|~0.7.0` (the latter of which I personally prefer), but the original intention seems to have been verbosity, which the current syntax provides, no?

If anything, I'd prefer `~0.6.3|~0.7.0`, honestly.